Makefile

```
@@ -46,16 +66,35 @@ test-workflow-contracts: ## Validate the mutation-testing caller contract
 typecheck: ## Typecheck all workspace crates
 	RUSTFLAGS="-D warnings" $(CARGO) check --workspace --all-targets --all-features $(BUILD_JOBS)
 
-markdownlint: ## Lint Markdown files
+markdownlint: spelling ## Lint Markdown files
```

checkmake (0.3.2)

[warning] Target body for "markdownlint" exceeds allowed length of 5 lines (8).

(maxbodylength)